ts/Data/DataTableCore.ts

```diff
@@ -434,8 +412,11 @@ export default DataTableCore;
  * */
 
 /**
+ * A typed array.
+ * @typedef {Int8Array|Uint8Array|Uint8ClampedArray|Int16Array|Uint16Array|Int32Array|Uint32Array|Float32Array|Float64Array} Highcharts.TypedArray
+ * //**
  * A column of values in a data table.
- * @typedef {Array<boolean|null|number|string|undefined>} Highcharts.DataTableColumn
+ * @typedef {Array<boolean|null|number|string|undefined>|TypedArray} Highcharts.DataTableColumn
```

Comment:
Why does this not contain the Array like our internal typing?

Reply:
Because this doclet is about `DataTableCore` which is only used in Highcharts Core where Array as a cell type has no use case. Do you have any suggestions on how this could be handled better way? Extension of `column` field's type only in the `DataTable` class?